.github/workflows/tests.yml

```diff
@@ -32,6 +32,7 @@ jobs:
         uses: eregon/use-ruby-action@master
```

Comment:
Could you switch to `uses: ruby/setup-ruby@v1` ?

Reply:
Sure. Whats the difference?

Reply:
https://github.com/ruby/setup-ruby/releases/tag/v1.13.0

Also `@v1` makes sure to only use code intended to be released :)